src/bin/ion/commands/generate/model.rs

```diff
@@ -179,6 +190,32 @@ impl TryFrom<&Value> for FullyQualifiedTypeReference {
     }
 }
 
+impl Display for FullyQualifiedTypeReference {
```

Comment:
Using `Display` here might not be a good idea, since this display implementation is particularly related to how TypeScript prints namespaces instead we already have a method called `string_representation` which uses `L::namespace_separator()` to print namespace and is also then used by tera to get the string representation of fully qualified namespace.
You can just add namespace separator for typescript in `Language` implementation of `TypescriptLanguage`. This way its consistent with other languages and easy to expand more languages in future.